float64 my_x
float64 my_y
int8 my_id
bool has_id
---
float64[] x
float64[] y
int8 id